sdk/mixedreality/azure-mixedreality-remoterendering/azure/mixedreality/remoterendering/_remote_rendering_client.py

```diff
@@ -0,0 +1,368 @@
+    lifetime of remote rendering sessions.
+
+    :param str remote_rendering_endpoint:
+        The rendering service endpoint. This determines the region in which the rendering session is created and
+        asset conversions are performed.
+    :param str account_id: The Azure Remote Rendering account identifier.
+    :param str account_domain:
+        The Azure Remote Rendering account domain. For example, for an account created in the eastus region, this
+        will have the form "eastus.mixedreality.azure.com"
+    :param credential: Authentication for the Azure Remote
+        Rendering account. Can be of the form of an AzureKeyCredential, TokenCredential or an AccessToken acquired
+        from the Mixed Reality Secure Token Service (STS).
+    :type credential: Union[TokenCredential, AzureKeyCredential, AccessToken]
+    """
+
+    def __init__(self, remote_rendering_endpoint, account_id, account_domain, credential, **kwargs):
+        # type: (str, str, str, Union[TokenCredential, AzureKeyCredential, AccessToken], Any) -> None
+
+        # TODO: verify the above type annotation is correct
+        # TODO: should credential be called credential because of AccessToken
+        if not remote_rendering_endpoint:
+            raise ValueError("remote_rendering_endpoint can not be None")
+
+        if not account_id:
+            raise ValueError("account_id can not be None")
+
+        if not account_domain:
+            raise ValueError("account_domain can not be None")
+
+        if not credential:
+            raise ValueError("credential can not be None")
+
+        if isinstance(credential, AccessToken):
+            pipeline_credential = StaticAccessTokenCredential(credential)
+
+        if isinstance(credential, AzureKeyCredential):
+            pipeline_credential = MixedRealityAccountKeyCredential(
+                account_id=account_id, account_key=credential)
+
+        endpoint_url = kwargs.pop(
+            'authentication_endpoint_url', construct_endpoint_url(account_domain))
+
+        # otherwise assume it is a TokenCredential and simply pass it through
+        pipeline_credential = get_mixedreality_credential(
+            account_id=account_id, account_domain=account_domain, credential=credential, endpoint_url=endpoint_url)
+        authentication_policy = BearerTokenCredentialPolicy(
+            pipeline_credential, endpoint_url + '/.default')
+
+        self._account_id = account_id
+
+        self._client = RemoteRenderingRestClient(
+            base_url=remote_rendering_endpoint,
+            authentication_policy=authentication_policy,
+            sdk_moniker=SDK_MONIKER,
+            **kwargs)
+
+# TODO: remove notes for where i got the code from
+# look at communication\azure-communication-administration\azure\communication\administration\
+# _phone_number_administration_client.py
+
+    def begin_asset_conversion(self, conversion_id, conversion_options, **kwargs):
```

Comment:
i will push back on this - if we create the conversion_id internally and return a LROPoller from the method there is no way for the customer to know what the ongoing conversion/spun up session ID is. They cannot retrieve it from the LROPoller as far as i can tell. without this customers who want to not poll are out of an option to investigate the conversion/session and interact with it (e.g. stop a session because it is not needed anymore)